subprojects/robotpy-wpiutil/gen/RawFrame.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+enums:
+  WPI_PixelFormat:
+    ignore: true
```

Comment:
Guessing not worth exposing this enum to Python, since there's an equivalent C++ enum in `cscore`?

Reply:
As I read it yes, the cscore side enum is what the rest of cscore uses in its public api, not the RawFrame version.